### DISCUSSION TITLE
>30 Day Expiration into --debug 2

### DISCUSSION THREAD

**Discussion by daBee** [acmesh-official/acme.sh#4976] [Ideas]
I tripped over ZeroSSL's indication that a certificate cannot be renewed if it has expired 30+ days before current.  

[https://help.zerossl.com/hc/en-us/articles/360060119813-Renewing-an-SSL-Certificate](https://help.zerossl.com/hc/en-us/articles/360060119813-Renewing-an-SSL-Certificate)

Instead of troubleshooting perfectly good manual attempts at renewing certificates that they have invalidated, how about testing the date of expiry and provide some proper feedback?

**Please note that a certificate can only be renewed within 30 days of its expiration date**

Cheers

**Comment by neilpang** [comment]
No,  I just forcely renewed one of my zerossl cert, which has more than 30 days left.

please confirm with the zerossl support whether a rule is applied to their webUI or the acme api first before post any thing here.

**Reply by daBee** [reply]
A rule?  It's an article in their documentation.  None of this is API nor webUI.  It's on the second page of **Getting Started**.
